Rakefile

```diff
@@ -10,6 +10,7 @@ Rake::TestTask.new do |t|
   t.libs.push "lib"
   t.test_files = Dir.glob('test/**/*_test.rb')
   t.verbose = true
+  t.warning = false unless ENV['RUBY_WARNINGS']
```

Comment:
Why default to false? Isn't it good to see warnings?

Reply:
It's good I guess, but most of the time we won't do anything about them (or can't due to warning coming from some library). It will also require to update any places we call `rake test` with `RUBY_WARNINGS=false` in that case.